src/yesql/generate.clj

```diff
@@ -72,21 +72,27 @@
 ;; case, we only ever use one group, so we'll unpack the
 ;; single-element list with `first`.
 (defn execute-handler
-  [db sql-and-params call-options]
-  (first (jdbc/execute! db sql-and-params)))
+  [db sql params call-options]
```

Comment:
yes. I had to do it this way because the parameters that were being sent in were being modified before the function call.